build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -202,55 +190,73 @@ rubygem-rake:
-simp-vendored-r10k-2.2.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-2.2.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-2.2.2-0.noarch.rpm
-simp-vendored-r10k-doc-2.2.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-doc-2.2.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-doc-2.2.2-0.noarch.rpm
-simp-vendored-r10k-gem-colored-1.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-colored-1.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-colored-1.2-0.noarch.rpm
-simp-vendored-r10k-gem-cri-2.6.1-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-cri-2.6.1-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-cri-2.6.1-0.noarch.rpm
-simp-vendored-r10k-gem-faraday-0.9.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-faraday-0.9.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-faraday-0.9.2-0.noarch.rpm
-simp-vendored-r10k-gem-faraday_middleware-0.10.1-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-faraday_middleware-0.10.1-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-faraday_middleware-0.10.1-0.noarch.rpm
-simp-vendored-r10k-gem-log4r-1.1.10-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-log4r-1.1.10-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-log4r-1.1.10-0.noarch.rpm
-simp-vendored-r10k-gem-minitar-0.6.1-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-minitar-0.6.1-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-minitar-0.6.1-0.noarch.rpm
-simp-vendored-r10k-gem-multi_json-1.12.1-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-multi_json-1.12.1-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-multi_json-1.12.1-0.noarch.rpm
-simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-multipart-post-2.0.0-0.noarch.rpm
-simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-puppet_forge-2.1.5-0.noarch.rpm
-simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm:
-  :rpm_name: simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm
-  :source: https://packagecloud.io/simp-project/6_X/packages/el/7/simp-vendored-r10k-gem-r10k-2.2.2-0.noarch.rpm
+simp-vendored-r10k:
+  :rpm_name: simp-vendored-r10k-2.6.2-1.noarch.rpm
+  :source: https://download.simp-project.com/simp/yum/simp6/el/7/x86_64/simp-vendored-r10k-2.6.2-1.noarch.rpm
```

Comment:
Seems inconsistent that the simp-vencored-r10k* packages are from packagecloud for el7, but download.simp-project.com here.

Reply:
Fixed